### PR TITLE
docs(transform): control

### PR DESCRIPTION
**Pull request by CristianSpatari**

### CODE REVIEW

File: docusaurus/docs-internals/editor-controls/data-controls/transform.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+| `devices?`                | `"all"` \| `"desktop"` \| `"responsive"`                                                                                                                                        | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+| `config?.disabled`        | `boolean`                                                                                                                                                                       | `false` | Configure the condition under which the control is disabled or enabled.                                                                                                                                                           |
+
+### Return value
+
+The return value is a `string`.
+```js
+{
+  value: string;
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/data-controls/transform.md:30]: why here is only 1 key is transform returns many keys ?

File: docusaurus/docs-internals/editor-controls/data-controls/transform.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+| `devices?`                | `"all"` \| `"desktop"` \| `"responsive"`                                                                                                                                        | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+| `config?.disabled`        | `boolean`                                                                                                                                                                       | `false` | Configure the condition under which the control is disabled or enabled.                                                                                                                                                           |
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/data-controls/transform.md:23]: about `config.disabled`

this is another props, not `disabled` from every option

`config.disabled` from your control is an array

File: docusaurus/docs-internals/editor-controls/data-controls/transform.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+| `id`                      | `string`                                                                                                                                                                        |    -    | The identifier of the key where the inputText will save your data                                                                                                                                                                 |
+| `type`                    | `string`                                                                                                                                                                        |    -    | Type should be `"inputText"` to use this control                                                                                                                                                                                  |
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/data-controls/transform.md:19]: about "Type should be `"inputText"` to use this control"

why inputText ? set your control name

File: docusaurus/docs-internals/editor-controls/data-controls/transform.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+#### Disabled examples
+
+Control will be disabled. Normally, here should be your dynamic condition.
+
+```js
+{
+  id: "effect",
+  type: "transform",
+  disabled: true
+}
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/data-controls/transform.md:100]: only this example ?

**Review by victor-pg — CHANGES_REQUESTED** [review]

File: docusaurus/static/img/data-controls/transform.png
Comment (victor-pg) [inline docusaurus/static/img/data-controls/transform.png:1]: move it to multivalue folder

**Review by victor-pg — CHANGES_REQUESTED** [review]
1. Move it to multivalue controls
image
2. image
3. image
4. image
5.

**Review by victor-pg — CHANGES_REQUESTED** [review]
Need to describe it as how described @grandleemon in `Motion`
image

File: docusaurus/docs-internals/editor-controls/multivalue-controls/transform.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+    "transformScaleScaleXY": undefined | number,
+    "transformScaleScalePreserveSize": undefined | boolean,
+
+    "transformFlipEnabled": undefined | boolean,
+    "transformFlipFlipHorizontal": undefined | boolean,
+    "transformFlipFlipVertical": undefined | boolean,
+
+    "transformAnchorPointX": undefined | string,
+    "transformAnchorPointY": undefined | string
+}
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/multivalue-controls/transform.md:60]: How look your return value: 
image

Make your return value and return value description look like in this PR: https://github.com/EasyBrizy/Brizy-Local-Editor/pull/259/files#diff-1f4f17d8adb93cf89a4dbfa84cd9eb9668f165d122ffefaf231f012982ab25bdR31-R46

**Review by victor-pg — CHANGES_REQUESTED** [review]
How your transform works with this effects ?

image

As example `mouseTilt`, how it supposed to be disabled if your transform has no `mouseTilt` property

**Review by victor-pg — CHANGES_REQUESTED** [review]
1. image
2. image

**Comment by CristianSpatari** [comment]
> 1. image
> 2. image

Because no disabled 
image

**Pull request merged**